src/Utils/StyleAttributesUtils.php

```diff
@@ -247,9 +266,30 @@ public static function get_border_radius_class_and_style( $attributes ) {
 			return null;
 		}
 
+		$border_radius_css = '';
+
+		if ( is_string( $custom_border_radius ) ) {
+			// Linked sides.
+			$custom_border_radius = 'border-radius:' . $custom_border_radius . ';';
```

Comment:
`border-radius` wasn't working on my end. I think it's because of this line. Should it be like this?
```suggestion
			$border_radius_css = 'border-radius:' . $custom_border_radius . ';';
```

Reply:
Thank you so much for the review and for pointing out the bugs.

That border color was super tough to nail down as the format of passed data varies greatly.